src/core/env/prod/terraform.tfvars

```diff
@@ -30,11 +30,14 @@ cidr_subnet_vpn                = ["10.0.133.0/24"]
 cidr_subnet_azdoa              = ["10.0.250.0/24"]
 cidr_subnet_dnsforwarder       = ["10.0.252.8/29"]
 
-app_gateway_api_certificate_name      = "api-io-pagopa-it"
-app_gateway_api_app_certificate_name  = "api-app-io-pagopa-it"
-app_gateway_api_mtls_certificate_name = "api-mtls-io-pagopa-it"
-app_gateway_min_capacity              = 0
-app_gateway_max_capacity              = 50
+app_gateway_api_certificate_name                                  = "api-io-pagopa-it"
+app_gateway_api_mtls_certificate_name                             = "api-mtls-io-pagopa-it"
+app_gateway_api_app_certificate_name                              = "api-app-io-pagopa-it"
+app_gateway_api_io_italia_it_certificate_name                     = "api-io-italia-it"
+app_gateway_app_backend_io_italia_it_certificate_name             = "app-backend-io-italia-it"
+app_gateway_developerportal_backend_io_italia_it_certificate_name = "developerportal-backend-io-italia-it"
+app_gateway_min_capacity                                          = 0
```

Comment:
is it safe to set min = 0 in production ?

Reply:
thanks @uolter I changed to 2 (now is not yet used, but probably tomorrow yes)
I also added logs to prod-sec (sentinel)